eparticlectr/eparticlectr.cpp

```diff
@@ -175,11 +54,39 @@ void eparticlectr::brainclmid( uint64_t stakeid ) {
     stake_it = staketable.erase(stake_it);
 }
 
+// Redeem IQ, with a specific stake specified
+[[eosio::action]]
+void eparticlectr::brainclmidex( uint64_t stakeid ) {
+
+    // Get the stakes
+    staketblex staketable(_self, _self.value);
+    auto stake_it = staketable.find(stakeid);
```

Comment:
not big deal, but as suggestion, you can use and simplify code:

```
const auto& stake_it = staketable.get(stakeid, "Stake does not exist in database");
```

Reply:
will change